app/views/info/team.html.erb

```diff
@@ -29,7 +29,7 @@ events, and political advocacy. In his spare time he enjoys local community orga
   </div>
 </div>
 <div class="bio">
-  <h2>JP Pantin</h2>
+  <h2 href = >JP Pantin</h2>

```

Comment:
https://www.linkedin.com/in/jp-pantin-98b560a7/

Reply:
added